### PR TITLE
Removed adapters

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Description
Removed usage of `adapters` outside test environment

## Issue link
closes https://github.com/networkservicemesh/sdk/issues/1147

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [x] Refactoring
- [ ] CI

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ linters-settings:
   dupl:
     threshold: 150
   funlen:
-    Lines: 100
+    Lines: 105
```
Comment (denis-tingaikin) [inline .golangci.yml:38]: Can we apply the fix for a file and not change this for all files?

File: pkg/networkservice/common/setlogoption/client.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		options: options,
+	}
+}
+
+func (s *setLogOptionClient) Request(ctx context.Context, request *networkservice.NetworkServiceRequest, opts ...grpc.CallOption) (*networkservice.Connection, error) {
+	ctx = s.withFields(ctx)
+	return next.Client(ctx).Request(ctx, request, opts...)
+}
+
+func (s *setLogOptionClient) withFields(ctx context.Context) context.Context {
```
Comment (denis-tingaikin) [inline pkg/networkservice/common/setlogoption/client.go:47]: Can we create a file context.go and put there function withFields?
https://github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/common/setlogoption/server.go#L46-L61

The function can be reused by client and server.

File: pkg/networkservice/common/setlogoption/client.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	ctxFields := log.Fields(ctx)
+	fields := make(map[string]interface{})
+	for k, v := range ctxFields {
+		fields[k] = v
+	}
```
Comment (denis-tingaikin) [inline pkg/networkservice/common/setlogoption/client.go:52]: Why do we need to make a copy?

File: .golangci.yml (hunk 1 above)
Comment (Mixaster995) [reply .golangci.yml]: ok, fixed

File: pkg/networkservice/common/setlogoption/client.go (hunk 2 above)
Comment (Mixaster995) [reply pkg/networkservice/common/setlogoption/client.go]: ok, fixed

File: pkg/networkservice/common/setlogoption/client.go (hunk 3 above)
Comment (Mixaster995) [reply pkg/networkservice/common/setlogoption/client.go]: ok, fixed

File: pkg/networkservice/common/setlogoption/context.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+)
+
+func withFields(ctx context.Context, options map[string]string) context.Context {
+	fields := make(map[string]interface{})
+	fields["type"] = "NetworkService"
+	for k, v := range options {
+		fields[k] = v
+	}
```
Comment (denis-tingaikin) [inline pkg/networkservice/common/setlogoption/context.go:30]: what if we already have fields in ctx?

**Comment by edwarnicke** [comment]
@Mixaster995 @denis-tingaikin Is there more work to do here or is this ready to merge?

**Comment by Mixaster995** [comment]
@edwarnicke it is ready, but i think it is better to be frozen until release, because any new change may affect stability, which is our №1 priority. What do you think about that?

File: pkg/networkservice/common/setlogoption/context.go (hunk 4 above)
Comment (Mixaster995) [reply pkg/networkservice/common/setlogoption/context.go]: ok, fixed

**Comment by denis-tingaikin** [comment]
@Mixaster995 , @edwarnicke  I think we actually do not need setlogoption pkg. Can we just cut it from sdk?

**Comment by Mixaster995** [comment]
closed in favor of https://github.com/networkservicemesh/sdk/pull/1186

**Pull request closed without merging**